### PR TITLE
Feature/task manager

### PR DESCRIPTION
**Pull request by ronjakr**
Created a task manager for switching between different autonomous tasks and manual drive (joystick)
Works the same way as task manager for Beluga

### CODE REVIEW

File: asv_setup/config/robots/temp_config.yaml [hunk 1]
```diff
@@ -94,3 +94,7 @@ joystick:
     roll: 35
     pitch: -30
     yaw: 20
+
+tasks:
+  joystick: True
+  collision_avoidance: False
```
Comment (theBadMusician) [inline asv_setup/config/robots/temp_config.yaml:100]: Remove if not used!

File: mission/task_manager/cfg/task_manager.cfg
Comment (theBadMusician) [inline mission/task_manager/cfg/task_manager.cfg:1]: Could be converted to python rcfg using https://github.com/pal-robotics/ddynamic_reconfigure_python framework.

File: mission/task_manager/package.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  <license>MIT</license>
+
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>dynamic_reconfigure</build_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>rospy</build_depend>
+  <build_export_depend>dynamic_reconfigure</build_export_depend>
+  <build_export_depend>roscpp</build_export_depend>
+  <build_export_depend>rospy</build_export_depend>
```
Comment (theBadMusician) [inline mission/task_manager/package.xml:20]: Are these needed? REMOVE!

File: mission/task_manager/scripts/task_manager_server_node.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+import rospy
+
+from dynamic_reconfigure.server import Server
+from task_manager.cfg import NjordTasksConfig
+
+
+def callback(config, level):
+    rospy.loginfo("""Server: task change request: {Njord_tasks}""".format(**config))
```
Comment (theBadMusician) [inline mission/task_manager/scripts/task_manager_server_node.py:9]: Remove loginfo printing or set it as debug condition.

File: mission/task_manager/task_manager_defines/defines.py
Comment (theBadMusician) [inline mission/task_manager/task_manager_defines/defines.py:1]: Document using comments in code pls!

File: mission/task_manager/test_nodes/collision_avoidance.py
Comment (theBadMusician) [inline mission/task_manager/test_nodes/collision_avoidance.py:1]: rename this to "template_node" to not be confusing

**Review by theBadMusician — CHANGES_REQUESTED** [review]
Was expecting better from you :(

File: mission/task_manager/scripts/task_manager_server_node.py (hunk 3 above)
Comment (ronjakr) [reply mission/task_manager/scripts/task_manager_server_node.py:9]: should maybe be warning, if this is useful information to get while the drone is operating?

**Review by theBadMusician — APPROVED** [review]

**Pull request merged**